src/lib/Scenes/Collection/Components/CollectionHeader.tsx

```diff
@@ -1,15 +1,39 @@
-import { Serif } from "@artsy/palette"
+import { Box, color, Serif } from "@artsy/palette"
+import OpaqueImageView from "lib/Components/OpaqueImageView/OpaqueImageView"
 import React from "react"
+import { Dimensions } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 
-export const CollectionHeader = props => {
-  const { title, description } = props.collection
+interface CollectionHeaderProps {
+  collection: {
```

Comment:
Instead of defining these props manually, we can import the generated types based on the fragment that's defined later on in the file, from the `__generated__` directory. Here's an example:

https://github.com/artsy/emission/blob/97727e4b07b7162fceb07e518ca255366f1dc5c1/src/lib/Scenes/Artwork/Components/ArtworkHeader.tsx#L2

Reply:
Good catch, just updated it!